flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -103,6 +103,8 @@
 	 */
 	private int maxParallelism = -1;
 
+	private String description;
```

Comment:
Because we should access it in JobGraphGenerator and StreamingJobGraphGenerator.We need to get it from the API in Batch and Streaming and pass it to JobGraph.

Reply:
If you implemented it the same way as jobName, you would have it there in the `OptimizedPlan`